test/views/declaration/DeclarationChoiceViewSpec.scala

```diff
@@ -117,16 +115,16 @@ class DeclarationChoiceViewSpec extends UnitViewSpec with CommonMessages with St
   }
   private def ensureAllLabelTextIsCorrect(view: Document): Unit = {
     view.getElementsByTag("label").size mustBe 5
-    view.getElementById("STANDARD-label").text() mustBe "declaration.type.standard"
-    view.getElementById("SUPPLEMENTARY-label").text() mustBe "declaration.type.supplementary"
-    view.getElementById("SIMPLIFIED-label").text() mustBe "declaration.type.simplified"
-    view.getElementById("OCCASIONAL-label").text() mustBe "declaration.type.occasional"
-    view.getElementById("CLEARANCE-label").text() mustBe "declaration.type.clearance"
+    view.getElementsByAttributeValue("for", "STANDARD").text() mustBe "declaration.type.standard"
```

Comment:
:+1: for attribute selector :)

Reply:
That's what we do in movements to select/test radio options